src/tools/tidy/src/deps.rs

```diff
@@ -28,6 +28,7 @@ const LICENSES: &[&str] = &[
     "Apache-2.0",
     "Apache-2.0/MIT",
     "BSD-2-Clause OR Apache-2.0 OR MIT",                   // zerocopy
+    "CC0-1.0",                                             // notify
```

Comment:
Who's responsible for this license exceptions list? Feels like it'd be good to check that these exceptions are OK with someone who's more experienced in this area.

Reply:
I know the right person for this. cc @wesleywiser :)